### PR TITLE
add conditions to use NWChem with Pynta 

### PR DESCRIPTION
**Pull request by sakim8048**
- Previously: Pynta uses QE as a default QM software. In main.py, the keywords in `__init__` are input keywords for QE.
- Modification: Keywords in __init__ are now all user-defined. In the input, user needs to define which QM software they want to use. (i.e. software = "NWChem"). Depending on the QM software that users define to use, pre-determined keywords ( `software_kwargs`) are also provided in Main.py accordingly. 

- Comments: Sample Pynta input with NWChem is attached. 

----------------------------------------------------------------------------------------------------------------------------
#this is a sample input for running nwchem. Notice that `software="NWChem"` is added. 

from pynta.main import Pynta
import sys
print(sys.prefix)

pyn = Pynta(path="/where/you/run/pynta/job",launchpad_path="/dir/to/your/my_launchpad.yaml",
        fworker_path="/dir/to/your/my_fworker.yaml",
        queue_adapter_path="/dir/to/your/my_qadapter.yaml",
        rxns_file="/dir/to/your/reaction.yaml",
                surface_type="fcc111",metal="Cu",`software="NWChem"`,socket=False,queue=True,njobs_queue=5,
        repeats=[(1,1,1),(3,3,4)],label="pynta-nwchem",
        
software_kwargs={'set nwpw': 'cif_filename debug',
                         'nwpw':{'smear':'marzari-vanderbilt',
                                  'ewald_ncut': 10,
                                  'ewald_rcut': 3.0,
                                  'xc':'beef-vdw',
                                  'loop':'10 10',
                                  'cutoff':10.0},
                                  'nwpw':{'pseudopotentials':'Cu library paw_default',
                                  '':'end'},
                            'set':{'nwpw:kbpp_ray': True,
                              'nwpw:kbpp_filter': True},
                "command": f'/script/to/run/nwchem  PREFIX.nwi PREFIX.nwo'},
        
software_kwargs_gas={'set nwpw': 'cif_filename debug',
                             'nwpw':{'smear':'marzari-vanderbilt',
                                  'ewald_ncut': 10,
                                  'ewald_rcut': 3.0,
                                  'xc':'beef-vdw',
                                  'kpts':(3,3,1),
                                  'loop': '10 10',
                                  'cutoff':10.0,
                                  'nwpw':{'pseudopotentials':'Cu library paw_default',
                                  '':'end'},
                                  },
                             'set':{'nwpw:kbpp_ray': True,
                                    'nwpw:kbpp_filter': True},
                "command": f'/script/to/run/nwchem  PREFIX.nwi PREFIX.nwo'},
        
TS_opt_software_kwargs=None,
        
lattice_opt_software_kwargs={'kpts': (3,3,1),
                                  'nwpw':{'pseudopotentials':'Cu library paw_default',
                                  '':'end'},
                              'set':{'nwpw:kbpp_ray': True,
                              'nwpw:kbpp_filter': True},
                "command": f'/script/to/run/nwchem  PREFIX.nwi PREFIX.nwo'},

slab_path = "/dir/to/your/slab.xyz",)
pyn.execute()

### CODE REVIEW

File: pynta/main.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def __init__(self,path,rxns_file,surface_type,metal,label,launchpad_path=None,fw
         lattice_opt_software_kwargs=None,
         reset_launchpad=False,queue_adapter_path=None,num_jobs=25,max_num_hfsp_opts=None,#max_num_hfsp_opts is mostly for fast testing
         Eharmtol=3.0,Eharmfiltertol=30.0,Ntsmin=5):
-
+# users need to define in input which QM software they want to use. Depending on the QM software the user define, default keywords are determined.
```
Comment (mjohnson541) [inline pynta/main.py:34]: This should be squashed in with the associated changes the comments apply to

**Review by mjohnson541 — CHANGES_REQUESTED** [review]
Thanks! I found some bugs and rebasing issues.

File: nersc-instructions [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#### this is a general instruction to install and launch pynta on nersc machines ###
```
[the remark this answers is not in the document]
Comment (sakim8048) [reply nersc-instructions]: I temporarily deleted this document for further revision. once more details are added, I will put the nersc section to the documentation.

File: pynta/main.py [hunk 3]
```diff
@@ -475,40 +476,24 @@ def launch(self):
         else:
             launch_multiprocess(self.launchpad,self.fworker,"INFO","infinite",self.num_jobs,5)
 
-    def execute(self,generate_initial_ad_guesses=True,calculate_adsorbates=True,
```
[the remark this answers is not in the document]
Comment (sakim8048) [reply pynta/main.py:478]: Thanks! yeah. I did not notice it back then. this time during the revision, I properly rebased on master and then further changes are made.

**Review by sakim8048 — COMMENTED** [review]
Requested changes are made. 

Summary:
- Rebased on master
- Unnecessary print statements and comments are deleted
- Default pseudopotential are defined only on Cu
- NWChem Cu pseudopotential is set as pspw_default
- Changed to elif statement for `software==nwchem`

**Pull request closed without merging**